lib/publiccloud/utils.pm

```diff
@@ -67,20 +67,21 @@ sub register_addon {
     my $timestamp = utc_timestamp();
     record_info($addon, "Going to register '$addon' addon\nUTC: $timestamp");
     my $cmd_time = time();
-    # ssh_add_suseconnect_product($remote, $name, $version, $arch, $params, $timeout, $retries, $delay)
+ # ssh_add_suseconnect_product($remote, $name, version => $version, arch => $arch, params => $params, timeout => $timeout, retries => $retries, delay => $delay)
```

Comment:
is indentation of this comment ok?

Reply:
This has been done by `perltidy` but I removed it now.